src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -1400,6 +1403,201 @@ public DatasetDTO getDatasetOfSampleList(final Integer sampleListId) {
+		}
+
+	}
+
+	private String createCountStudyQueryString(final StudySearchFilter studySearchFilter) {
+		final StringBuilder sql = new StringBuilder(" SELECT COUNT(DISTINCT geoloc.nd_geolocation_id) ");
+		this.appendStudySummaryFromQuery(sql);
+		this.appendStudySearchFilter(sql, studySearchFilter);
+		return sql.toString();
+	}
+
+	private String createStudySummaryQueryString(final StudySearchFilter studySearchFilter) {
+		final StringBuilder sql = new StringBuilder(" SELECT  ");
+		sql.append("     geoloc.nd_geolocation_id AS " + StudySearchFilter.STUDY_DB_ID + ", ");
+		sql.append("		CONCAT(pmain.name, ' Environment Number ', geoloc.description) AS " + StudySearchFilter.STUDY_NAME + ", ");
+		sql.append("     pmain.study_type_id AS " + StudySearchFilter.STUDY_TYPE_DB_ID + ", ");
+		sql.append("     studyType.label AS " + StudySearchFilter.STUDY_TYPE_NAME + ", ");
+		sql.append("     geopropSeason.value AS " + StudySearchFilter.SEASON_DB_ID + ", ");
+		sql.append("     cvtermSeason.definition AS " + StudySearchFilter.SEASON + ", ");
+		sql.append("     pmain.project_id AS " + StudySearchFilter.TRIAL_DB_ID + ", ");
+		sql.append(" 	 pmain.name AS " + StudySearchFilter.TRIAL_NAME + ", ");
+		sql.append("     MAX(pmain.start_date) AS " + StudySearchFilter.START_DATE + ", ");
+		sql.append("     MAX(pmain.end_date) AS " + StudySearchFilter.END_DATE + ", ");
+		sql.append("     CASE WHEN pmain.deleted = 0 THEN 1 ELSE 0 END AS " + StudySearchFilter.ACTIVE + ", ");
+		sql.append("     location.locid AS " + StudySearchFilter.LOCATION_DB_ID + ", ");
+		sql.append("     location.lname AS " + StudySearchFilter.LOCATION_NAME + ", ");
+		sql.append("     wp.project_name AS " + StudySearchFilter.PROGRAM_NAME + ", ");
+		sql.append("     wp.project_uuid AS " + StudySearchFilter.PROGRAM_DB_ID + " ");
+		this.appendStudySummaryFromQuery(sql);
+		this.appendStudySearchFilter(sql, studySearchFilter);
+		sql.append(" GROUP BY geoloc.nd_geolocation_id ");
+		if (!StringUtils.isEmpty(studySearchFilter.getSortedRequest().getSortBy()) && StudySearchFilter.SORTABLE_FIELDS
+			.contains(studySearchFilter.getSortedRequest().getSortBy())) {
+			sql.append(
+				" ORDER BY " + studySearchFilter.getSortedRequest().getSortBy() + " " + (
+					studySearchFilter.getSortedRequest().getSortOrder() == null ? "ASC" :
+						studySearchFilter.getSortedRequest().getSortOrder()));
+		}
+		return sql.toString();
+	}
+
+	private void appendStudySummaryFromQuery(final StringBuilder sql) {
+		sql.append(" FROM ");
+		sql.append("     nd_geolocation geoloc ");
+		sql.append("         INNER JOIN ");
+		sql.append("     nd_experiment nde ON nde.nd_geolocation_id = geoloc.nd_geolocation_id ");
+		sql.append("         INNER JOIN ");
+		sql.append("     project proj ON proj.project_id = nde.project_id ");
+		sql.append("         INNER JOIN ");
+		sql.append("     project pmain ON pmain.project_id = proj.study_id ");
+		sql.append("         LEFT OUTER JOIN ");
```

Comment:
It seems this join is unnecessary given you have separate joins for the season and location geolocationprop records: 
`LEFT OUTER JOIN nd_geolocationprop geoprop `